tests/package.json

```diff
@@ -16,8 +16,9 @@
     "colors": "^1.2.1",
     "cfenv": "^1.0.0",
     "kafka-node": "^2.4.0",
-    "@open-iot-service-platform/oisp-sdk-js": "^1.0.0-beta1",
     "nodemailer": "^4.6.6",
-    "imap": "^0.8.19"
+    "imap": "^0.8.19",
+    "mailparser": "^2.2.0",
+    "@open-iot-service-platform/oisp-sdk-js":"https://github.com/Open-IoT-Service-Platform/oisp-sdk-js.git#99d3db293f69ecd7e9ee6fab1afc132b9c1adc71"
```

Comment:
Change open-sdk-js hash to https://github.com/chuan9/oisp-sdk-js.git#fc6722efbe6cf5c7c2730b9cb0cce7cbb1ad69a1